services/ui/ws/platform_display_default.cc

```diff
@@ -66,13 +66,13 @@ EventSink* PlatformDisplayDefault::GetEventSink() {
   return delegate_->GetEventSink();
 }
 
-void PlatformDisplayDefault::Init(PlatformDisplayDelegate* delegate) {
+void PlatformDisplayDefault::Init(PlatformDisplayDelegate* delegate,
+                                  PlatformDisplay* parent_display) {
```

Comment:
can we access parent from PlatformDisplayDelegate instead, not avoid changing the signature of the method?